fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1275,6 +1294,26 @@ ACTOR static Future<Void> readRequestServer(
 	}
 }
 
+ACTOR Future<Void> healthMetricsRequestServer(MasterProxyInterface proxy, HealthMetrics* healthMetrics, GetDetailedHealthMetricsReply* getDetailedHealthMetricsReply)
+{
+	loop {
+		choose {
+			when(GetHealthMetricsRequest req =
```

Comment:
As mentioned in a different comment, consider combining these two promise streams into one